ui/hooks/gasFeeInput/useGasFeeErrors.js

```diff
@@ -21,6 +21,7 @@ import { GAS_FORM_ERRORS } from '../../helpers/constants/gas';
 const HIGH_FEE_WARNING_MULTIPLIER = 1.5;
 
```

Comment:
After creating context over `useGasFeeHook` e2e was unstable in FF cox of race condition, these null checks are added for that.

Reply:
This is concerning. Why would `gasLimit` be `undefined`? I see that it is supposed to be initialized to zero in `useGasFeeInputs`. Maybe I'm missing a case.

Reply:
`useState` hook is asynchronous, it is less likely but possible that hook to find errors is executed at same time.

For other fields utility methods used for comparison already have check for `undefined` value. But I found that these 3 fields did not had that: `gasLimit`, `minimumCostInHexWei`, `ethBalance`. May be `gasLimit` is not reason here, I can try to remove null check for it and see.

Reply:
From last commit it seems `gasLimit` was not the reason.

Reply:
`useState` is definitely synchronous. Everything that happens during the render is synchronous, so it will get initialized. If something is set to `undefined`, it must mean it was not initialized properly in the first place, or it was set to `undefined`.

Reply:
Yep render is definitely synchronous, but I was thinking of this possibility as error code that was throwing exception is run before render. But I am probably wrong here.